versions.json

```diff
@@ -360,14 +364,15 @@
         "fpm_directories": []
     },
     "qpid-proton": {
-        "commitish": "0.23.0",
-        "version_string": "0.23.0",
+        "github_repo": "https://github.com/apache/qpid-proton",
+        "commitish": "0.36.0",
+        "version_string": "0.36.0",
```

Comment:
requires pulling in latest changes from upstream.

Reply:
i've updated irods/qpid-proton, it's now in sync with upstream main.

Reply:
removed `"github_repo"`